packages/frontapp/src/app/types/materialsTypes.ts

```diff
@@ -0,0 +1,21 @@
+export interface IStatus {
+  __typename?: 'Status';
+  id: string;
+  created_at: any;
+  status: string;
+}
+
+export interface IMaterial {
+  __typename?: 'Material';
+  title: string;
+  created_at: any;
+  picture?: string | null;
+  author: string;
+  category: string;
+  id: string;
+  statuses: Array<IStatus | null>;
+}
```

Comment:
Use Pick or Omit from already declared type in apollo-client/../generated/client.types.ts